ui/tests/integration/components/app-form/project-settings-test.ts

```diff
@@ -0,0 +1,24 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, clearRender, fillIn } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | app-form/project-settings', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('second new project does not have previous input', async function (assert) {
+    this.set('project', {});
+    await render(hbs`<AppForm::ProjectSettings @project={{this.project}} />`);
+    await fillIn('#git-source-url', 'https://github.projectone.git');
+    await fillIn('#git-source-username', 'admin');
+    await fillIn('#git-source-password', 'password');
+    await clearRender();
+
+    this.set('project2', {});
+    await render(hbs`<AppForm::ProjectSettings @project={{this.project2}} />`);
+
+    assert.dom('#git-source-url').hasValue('');
+    assert.dom('#git-source-username').hasValue('');
+    assert.dom('#git-source-password').hasValue('');
```

Comment:
This test is good, but considering the changes above to the default data & `authCase` logic, I'd like to add a test with existing `dataSource` data covering both ssh and basic settings to verify that everything still works. I'm happy to work on adding that test or to pair with you tomorrow on that.